app/code/Magento/Sitemap/view/adminhtml/templates/js.phtml

```diff
@@ -0,0 +1,12 @@
+<script>
```

Comment:
Hi @Bartlomiejsz, thanks for collaboration. 
Please use `[x-magento-init](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/javascript/js_init.html)` construction instead of defining javascript directly in `phtml` file.

Reply:
Hi @VladimirZaets, done